src/main/resources/templates/questions/BBSIQ.html

```diff
@@ -74,12 +75,12 @@ <h2 class="page-title heading-icon clearfix"> <img th:src="@{/images/map-arrow-o
                                 </tr>
                                 <tr>
                                     <td class="statement">They did not enjoy the visit and were bored with your company.</td>
-                                    <td><label class="radioBox"><input type="radio" name="visitors_bored" class="form-control" value="0" required="true"/>0</label></td>
-                                    <td><label class="radioBox"><input type="radio" name="visitors_bored" class="form-control" value="1" required="true"/>1</label></td>
-                                    <td><label class="radioBox"><input type="radio" name="visitors_bored" class="form-control" value="2" required="true"/>2</label></td>
-                                    <td><label class="radioBox"><input type="radio" name="visitors_bored" class="form-control" value="3" required="true"/>3</label></td>
-                                    <td><label class="radioBox"><input type="radio" name="visitors_bored" class="form-control" value="4" required="true"/>4</label></td>
-                                    <td><label class="radioBox"><input type="radio" name="visitors_bored" class="form-control" value="555" required="true"/></label></td>
+                                    <td><label class="radioBox" id="last"><input type="radio" name="visitors_bored" class="form-control" value="0" required="true"/>0</label></td>
```

Comment:
Surprised this is working.  The ID should be unique within the entire html page.  You should never have a list of items all with the same id.  Even if this works in some browsers it will certainly break in others.  http://www.w3schools.com/tags/att_global_id.asp   Definitely need a different approach here.